controllers/products.js

```diff
@@ -1,16 +1,8 @@
-//products controller
-
-
-// GET products
-// GET product by id
-
-
+import Products from "../data/products.js";
 
 // POST product
-
-
-// UPDATE product
-
-// DELETE product
-
-
+export const addProduct = (req, res) => {
+  const id = Products.length;
+  Products.push({ ...req.body, id });
+  res.send({ ...req.body, id });
```

Comment:
For success you should return a status code of 200.

res.status(200)

Reply:
done